packages/libs/eda/src/lib/map/analysis/hooks/standaloneMapMarkers.tsx

```diff
@@ -0,0 +1,598 @@
+      : overlayType === 'continuous'
+      ? markerVariableBundle?.overlayConfig?.overlayValues.map(
+          (ov) => ov.binLabel
+        )
+      : undefined;
+
+  const markerData = usePromise<StandaloneMapMarkersResponse | undefined>(
+    useCallback(async () => {
+      // check all required vizConfigs are provided
+      if (
+        boundsZoomLevel == null ||
+        geoConfig == null ||
+        latitudeVariable == null ||
+        longitudeVariable == null ||
+        geoAggregateVariable == null ||
+        markerVariableBundle?.outputEntityId == null
+      )
+        return undefined;
+
+      const {
+        northEast: { lat: xMax, lng: right },
+        southWest: { lat: xMin, lng: left },
+      } = boundsZoomLevel.bounds;
+
+      // now prepare the rest of the request params
+      const requestParams: StandaloneMapMarkersRequestParams = {
+        studyId,
+        filters: filters || [],
+        config: {
+          geoAggregateVariable,
+          latitudeVariable,
+          longitudeVariable,
+          ...markerVariableBundle, // also includes outputEntityId
+          valueSpec: 'count', // TO DO: or proportion when we have the UI and back-end fix https://github.com/VEuPathDB/EdaDataService/issues/261 for this
+          viewport: {
+            latitude: {
+              xMin,
+              xMax,
+            },
+            longitude: {
+              left,
+              right,
+            },
+          },
+        },
+      };
+
+      // now get the data
+      return await dataClient.getStandaloneMapMarkers(
+        computationType,
+        requestParams
+      );
+    }, [
+      studyId,
+      filters,
+      dataClient,
+      markerVariableBundle,
+      geoAggregateVariable,
+      latitudeVariable,
+      longitudeVariable,
+      boundsZoomLevel,
+      computationType,
+      geoConfig,
+    ])
+  );
+
+  const totalVisibleEntityCount: number | undefined =
+    markerData.value?.mapElements.reduce((acc, curr) => {
+      return acc + curr.entityCount;
+    }, 0);
+
+  // calculate minPos, max and sum for chart marker dependent axis
+  // assumes the value is a count! (so never negative)
+  const { valueMax, valueMinPos, valueSum } = useMemo(
+    () =>
+      markerData.value
+        ? markerData.value.mapElements
+            .flatMap((el) => el.overlayValues)
+            .reduce(
+              ({ valueMax, valueMinPos, valueSum }, elem) => ({
+                valueMax: elem.value > valueMax ? elem.value : valueMax,
+                valueMinPos:
+                  elem.value > 0 &&
+                  (valueMinPos == null || elem.value < valueMinPos)
+                    ? elem.value
+                    : valueMinPos,
+                valueSum: (valueSum ?? 0) + elem.value,
+              }),
+              {
+                valueMax: 0,
+                valueMinPos: undefined as number | undefined,
+                valueSum: undefined as number | undefined,
```

Comment:
Actually we want the return `valueSum` to be nullish if there is no overlay (the back end response `mapElements[].overlayValues` is an empty array in this case.

Because `totalVisibleWithOverlayEntityCount = valueSum` we want that to be nullish when it gets sent to the map header, so it can fall back to the other sum from actual entity counts.